docs/compression_algorithms/CompressWeights.md

```diff
@@ -24,26 +23,201 @@ from nncf import compress_weights
 compressed_model = compress_weights(model)
 ```
 
-- Compress weights to nf4 data type with group size = 128, except first embedding and last linear layers - they are compressed to 8-bit integer data type.
+- Compress weights symmetrically to 4-bit integer data type with group size = 128, except first embedding and last linear layers - they are compressed to 8-bit integer data type.
 
 ```python
 from nncf import compress_weights
 from nncf import CompressWeightsMode
-compressed_model = compress_weights(model, mode=CompressWeightsMode.NF4)
+compressed_model = compress_weights(model, mode=CompressWeightsMode.INT4_SYM)
 ```
 
-- Compress weights of 90% of layers to nf4 with the group size 64, and the rest of layers to 8-bit integer data type.
+- Generally, `INT4_SYM` mode is the fastest mixed-precision mode, but it may lead to a significant accuracy degradation or perplexity increase.
+  Compressing weights asymmetrically (`INT4_ASYM` mode) is the way to increase accuracy, however in turns it slows down inference a bit.
+  If the accuracy or perplexity is still not satisfying, there are 2 more hyper-parameters to tune: `group_size` and `ratio`.
```

Comment:
As the user I'd like to know which option should be tried first. Is it group_size? Do we have any recommendation here?